server/routes/api/dashboard.js

```diff
@@ -0,0 +1,26 @@
+const router = require("express").Router();
+const controller = require("../../controllers");
+
+// Matches with "/api/dashboard"
+router
+  .route("/:id")
+  .get(controller.getDashboard)
+  .put(controller.updateChallenge)
+
+router
+  .route("/:id/newchallenge")
+  .get(controller.getChallenge)
+
+router
+  .route("/:id/movie")
+  .get(controller.getMovie)
+
+router
+  .route("/:id/physical")
+  .get(controller.getPhysAct)
+
+router
+  .route("/:id/mental")
+  .get(controller.getMentalAct)
+
```

Comment:
@ShepLT1 can you add comments for where these should be used/ by which component on the frontend?